system/react/src/components/Toggle.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import { Story } from '@storybook/react';
+import * as React from 'react';
+
+import { classySelector, Toggle } from './Toggle';
+
+const contentVariants = ['Default', 'Disabled', 'Hover'] as const;
+
+export default {
+  title: 'TableKit/Toggle',
+  component: Toggle,
+  parameters: {
+    classySelector,
+    variants: contentVariants
+  }
+};
+
+export const AllVariants: Story = () => {
+  const refs1 = React.useRef<React.RefObject<HTMLInputElement>[]>([
+    React.createRef(),
+    React.createRef(),
+    React.createRef()
+  ]);
+  const refs2 = React.useRef<React.RefObject<HTMLInputElement>[]>([
+    React.createRef(),
+    React.createRef(),
+    React.createRef()
+  ]);
+
+  React.useEffect(() => {
+    refs1.current.forEach((ref) => {
+      if (ref.current) ref.current.indeterminate = true;
+    });
+    refs2.current.forEach((ref) => {
+      if (ref.current) ref.current.indeterminate = true;
+    });
+  }, []);
+
+  return (
+    <>
+      {[false, true].map((state) =>
+        [undefined, 'small'].map((size) =>
+          contentVariants.map((variant) => (
+            <Toggle
+              data-pseudo={variant.toLowerCase()}
+              data-size={size}
+              disabled={variant === 'Disabled'}
+              checked={state}
+            />
+          ))
+        )
+      )}
+      {contentVariants.map((variant, index) => (
+        <Toggle
+          data-pseudo={variant.toLowerCase()}
+          data-size={undefined}
+          disabled={variant === 'Disabled'}
+          ref={refs1.current[index]}
+        />
+      ))}
+      {contentVariants.map((variant, index) => (
+        <Toggle
+          data-pseudo={variant.toLowerCase()}
+          data-size="small"
+          disabled={variant === 'Disabled'}
+          ref={refs2.current[index]}
```

Comment:
This is simpler and needs less code.
```suggestion
          ref={(ref) => { if (ref) ref.indeterminate = true; }}
```

Reply:
thank you! done